custom/icds_reports/management/commands/imbalanced_indicator.py

```diff
@@ -0,0 +1,132 @@
+from __future__ import absolute_import, print_function
+from __future__ import unicode_literals
+
+from django.core.management.base import BaseCommand
+
+import dateutil
+from corehq.sql_db.connections import get_icds_ucr_db_alias
+from django.db import connections
+
+
+class Command(BaseCommand):
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            'input_month'
+        )
+
+    def handle(self, input_month, *args, **options):
+        indicator_list = self.get_indicator_list()
+        indicator_ratio = []
+
+        for indicator in indicator_list:
+            input_month = dateutil.parser.parse(input_month).date().replace(day=1).__str__()
+            query = self.prepare_query(indicator, input_month)
+            ratio_check = self.execute_query(query)
+            ratio_check = [(indicator["indicators"][i]['numerator'], res) for i, res in enumerate(ratio_check)]
```

Comment:
It would be great if this had the indicator name instead of the numerator which will be way easier to read and determine exactly what is off, instead of trying to match a numerator to an indicator.